v-next/hardhat/test/internal/cli/main.ts

```diff
@@ -57,7 +62,199 @@ async function getTasksAndHreEnvironment(
   };
 }
 
+async function getTasksAndSubtaskResults(
+  configFileName: string = "hardhat.config.ts",
+): Promise<boolean[]> {
+  // To ensure that one or more tasks have been executed, each task will modify an array of boolean values, initially set to false.
+  // This function imports that array, allowing the tests to verify if the tasks have been executed.
+  // If a boolean flag is true, it indicates that the corresponding task (or a specific part of it) has been executed.
+  // The array is set in the hardhat.config.ts file of the fixture project.
+  return (
+    await import(pathToFileURL(`${process.cwd()}/${configFileName}`).toString())
+  ).results;
+}
+
 describe("main", function () {
+  describe("main", function () {
```

Comment:
This `describe("main", ...` is duplicated

Reply:
It's ugly, but it's expected. Because the name of the file that I'm testing is "main" and the function that I'm testing is "main" too